src/mysql_dirver/impls/mysql_repositories.py

```diff
@@ -0,0 +1,52 @@
+from src.core.entities.customer_entitiy import Customer
+from src.core.repositories import CustomerRepository
+from src.core.repositories import EventRepository
+from src.core.repositories import OrderRepository
+from src.mysql_dirver.impls.cqrs.queries import get_all_customers
+
+
+class CustomerMySQLRepository(CustomerRepository):
+    def get_all(self):
+        customers = get_all_customers()
+
+        customer_entities = []
+        for customer in customers:
```

Comment:
Please, make a generator here. We are assuming we will make tests on very huge amount of data, so pushing everything into memory is not a good idea.